makefile

```diff
@@ -0,0 +1,5 @@
+all:
+	@ cd server \
+		&& govendor sync \
```

Comment:
**No!**

1. It's batter to make container as thin as it possible, so adding useless dependencies such as `git` and/or `govendor` is anti-pattern
2. You may have some private repositories in requirements list. If so, you are able to pass some access credentials to the container, which could cause the security leaks.